### DISCUSSION TITLE
wildcard subdomains on vercel.app/now.sh domains?

### DISCUSSION THREAD

**Discussion by maximousblk** [vercel/vercel#5159] [Help] [answered]
say I have a production deployment at `myapp.now.sh`, can I use wildcard domains on that alias by using `*.myapp.now.sh`?

I tried doing that but it just said that the alias is already in use. Is there any other way to do it?

my use case is very similar to the [static.fun](https://static.fun) demo.

**Comment by pg07codes** [comment]
@maximousblk did you find out a possible solution to this ?

**Reply by maximousblk** [reply]
I don't think this is possible as of now but there is no clear statement denying it.

We decided to buy a domain for our project so, no I didn't find the solution yet.

**Comment by wiiiimm** [comment, marked as answer]
@pg07codes @maximousblk Vercel only supports wildcards on custom domains. You can buy a domain from Vercel or from another domain registrar.
